### PR TITLE
Variety Changes

### PR DESCRIPTION
**Pull request by Churro55**
-Fixed the Aerostatic boots sprite so that it fits the inventory better. Allowed it to store a knife.
-Blacksilk boots, longboots, redboots, all cowboy boots and all winter boots can now hold knives too.
-Changed armor values of coats because some where wrong and others were being powercrept (again)
-Avasarala, commonly known as the worst of the .50 pistols, also had an absolutely PITYFUL firing sound. Gave it a more meaty one due to it being an absolute hunk of metal, and a slight one hand penalty reduction.
-Fixed the Trauma Team's Formatbound sprite so it also alligns better with the inventory (hopefully)
-Nanoregen power cost set to 7.
-Light typo fixes.

## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/projectile/pistol/avasarala.dm [hunk 1]
```diff
@@ -14,10 +14,11 @@
 	damage_multiplier = 1
 	penetration_multiplier = 0.9
 	recoil_buildup = 8
-	one_hand_penalty = 15
+	one_hand_penalty = 12
 	unload_sound 	= 'sound/weapons/guns/interact/hpistol_magout.ogg'
 	reload_sound 	= 'sound/weapons/guns/interact/hpistol_magin.ogg'
 	cocked_sound 	= 'sound/weapons/guns/interact/hpistol_cock.ogg'
+	fire_sound = 'sound/weapons/guns/fire/fire_silver.ogg'
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/projectile/pistol/avasarala.dm:21]: Do not do this

**Review by Trilbyspaceclone — CHANGES_REQUESTED** [review]
format_bound0 sprite isnt alined so it moves up and down

File: code/modules/projectiles/guns/projectile/pistol/avasarala.dm (hunk 1 above)
Comment (Churro55) [reply code/modules/projectiles/guns/projectile/pistol/avasarala.dm:21]: Why

Comment (cdb-is-not-good) [reply code/modules/projectiles/guns/projectile/pistol/avasarala.dm:21]: the silver-eye sound effect? eh...the Ava is intended to be a poorly performing gun that just looks cool, no?

Comment (Trilbyspaceclone) [reply code/modules/projectiles/guns/projectile/pistol/avasarala.dm:21]: Gun sounds that are uniquic to a gun should only be on that gun, in cases were people hear it to have better hear witnesses

Comment (Churro55) [reply code/modules/projectiles/guns/projectile/pistol/avasarala.dm:21]: Except for a literal single gun and the recent Vector, no other gun in the entire server has ever had unique sounds

Comment (Churro55) [reply code/modules/projectiles/guns/projectile/pistol/avasarala.dm:21]: Fixed the format_bound0 sprite. How do I add it to this commit?

Comment (Trilbyspaceclone) [reply code/modules/projectiles/guns/projectile/pistol/avasarala.dm:21]: click push on github desktop
were it has the little arrow with the 1

Comment (Trilbyspaceclone) [reply code/modules/projectiles/guns/projectile/pistol/avasarala.dm:21]: Also many guns have uniquic sounds.
Silver eye will keep its own sound as its own.

**Review by Trilbyspaceclone — APPROVED** [review]
Looks all good

File: code/modules/projectiles/guns/projectile/pistol/avasarala.dm [hunk 2]
```diff
@@ -14,10 +14,11 @@
...
 	penetration_multiplier = 0.9
 	recoil_buildup = 8
-	one_hand_penalty = 15
-	unload_sound 	= 'sound/weapons/guns/interact/hpistol_magout.ogg'
+	one_hand_penalty = 12
+	unload_sound 	= 'sound/weapons/guns/interact/avasarala_unload.ogg'
 	reload_sound 	= 'sound/weapons/guns/interact/hpistol_magin.ogg'
-	cocked_sound 	= 'sound/weapons/guns/interact/hpistol_cock.ogg'
+	cocked_sound 	= 'sound/weapons/guns/interact/avasarala_cock.ogg'
+	fire_sound 		= 'sound/weapons/guns/fire/avasarala_fire.ogg'
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/projectile/pistol/avasarala.dm:21]:
```suggestion
	fire_sound 	= 'sound/weapons/guns/fire/avasarala_fire.ogg'
```

File: code/modules/projectiles/guns/projectile/pistol/avasarala.dm [hunk 3]
```diff
@@ -14,10 +14,11 @@
...
 	penetration_multiplier = 0.9
 	recoil_buildup = 8
-	one_hand_penalty = 15
-	unload_sound 	= 'sound/weapons/guns/interact/hpistol_magout.ogg'
+	one_hand_penalty = 12
+	unload_sound 	= 'sound/weapons/guns/interact/avasarala_unload.ogg'
 	reload_sound 	= 'sound/weapons/guns/interact/hpistol_magin.ogg'
-	cocked_sound 	= 'sound/weapons/guns/interact/hpistol_cock.ogg'
+	cocked_sound 	= 'sound/weapons/guns/interact/avasarala_cock.ogg'
+	fire_sound 	= 'sound/weapons/guns/fire/avasarala_fire.ogg'
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/projectile/pistol/avasarala.dm:21]:
```suggestion
	fire_sound = 'sound/weapons/guns/fire/avasarala_fire.ogg'
```

**Pull request merged**